pkg/apis/apps.kubermatic/v1/application_definition.go

```diff
@@ -35,25 +35,31 @@ const (
 type HelmCredentials struct {
 	// Username holds the ref and key in the secret for the username credential.
 	// The Secret must exist in the namespace where KKP is installed (default is "kubermatic").
-	// The Secret must be annotated with `apps.kubermatic.k8c.io/secret-type:` set to helm or git
+	// The Secret must be annotated with `apps.kubermatic.k8c.io/secret-type:` set to "helm" or "git"
 	Username *corev1.SecretKeySelector `json:"username,omitempty"`
 
-	// Password holds the ref and key in the secret for the Password credential.
+	// Password holds the ref and key in the secret for the password credential.
 	// The Secret must exist in the namespace where KKP is installed (default is "kubermatic").
-	// The Secret must be annotated with `apps.kubermatic.k8c.io/secret-type:` set to helm or git
+	// The Secret must be annotated with `apps.kubermatic.k8c.io/secret-type:` set to "helm" or "git"
 	Password *corev1.SecretKeySelector `json:"password,omitempty"`
 
-	// RegistryConfigFile holds the ref and key in the secret for the registry credential file. The value is dockercfg
-	// file that follows the same format rules as ~/.docker/config.json
-	// The The Secret must exist in the namespace where KKP is installed (default is "kubermatic").
-	// The Secret must be annotated with `apps.kubermatic.k8c.io/secret-type:` set to helm or git
+	// RegistryConfigFile holds the ref and key in the secret for the registry credential file.
+	// The value is dockercfg file that follows the same format rules as ~/.docker/config.json.
+	// The Secret must exist in the namespace where KKP is installed (default is "kubermatic").
+	// The Secret must be annotated with `apps.kubermatic.k8c.io/secret-type:` set to "helm" or "git"
 	RegistryConfigFile *corev1.SecretKeySelector `json:"registryConfigFile,omitempty"`
 }
 
 type HelmSource struct {
-	// URl of the helm repository.
-	// It can be an HTTP(s) repository (e.g. https://localhost/myrepo) or on OCI repository (e.g. oci://localhost:5000/myrepo).
-	// +kubebuilder:validation:Pattern="^(http|https|oci)://.+"
+	// +kubebuilder:validation:Pattern="^(http|https|oci|oci-insecure)://.+"
```

Comment:
I am no Regex-Expert, but I think this does not match "oci+insecure://localhost:5000/myrepo" or "oci+http://localhost:5000/myrepo"

I think it should be like this `^(http|https|oci|oci\+insecure|oci\+http)://.+`

Reply:
Oh right, I typo'ed.